src/textual/widget.py

```diff
@@ -91,6 +91,10 @@ def __init__(self, parent: Widget, widgets: Sequence[Widget]) -> None:
         self._parent = parent
         self._widgets = widgets
 
+    async def __call__(self) -> None:
```

Comment:
Is this so we can use `invoke` to await this?

Reply:
precisely.